release_notes/ocp-4-14-release-notes.adoc

```diff
@@ -167,6 +167,12 @@ For more information about this more flexible SR-IOV network deployment during N
 ==== Update to HAProxy 2.6
 With this release, {product-title} updated to HAProxy 2.6.
 
+[id="ocp-4-14-networking-changing-ovn-kubernetes-network-plugin-internal-range"]
+==== Support for changing the OVN-Kubernetes network plugin internal IP address range
+
+If you use the OVN-Kubernetes network plugin, the default subnets used are `169.254.169.0/29` and `fd69::/125` for IPv4 and IPv6 respectively. Now you can change these subnets on installed clusters. For more information, see xref:../networking/cluster-network-operator.adoc#nw-operator-cr-cno-object_cluster-network-operator[Cluster Network Operator configuration object].
+
```

Comment:
Can you add more on how to change subnet on installed cluster?  what oc command to use?

Reply:
@jechen0648, sure, that work is taking place in this PR: https://github.com/openshift/openshift-docs/pull/61109

Reply:
@jechen0648, so from what I can tell, this is not currently supported as a post-installation activity.

Reply:
@jechen0648 @jab-rh this is currently not supported post-install, the CNO change would not trigger a re-rollout of the nodes so no change would occur, there may be a way around that, but for now this is only supported at install time